src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -270,6 +270,7 @@ When migrating from our traditional Azure integrations to the Azure Monitor inte
 
 * When you enable an Azure Monitor integration, new separate entities will be created for all of your resources. The entities created by the Azure Polling integration are staying as they are. This means you must update dashboards, alerts, and any other capability that refrences those entities.
 * Old entities are available for 24 hours.
+* A metric name may appear to show up twice when it has different dimension combinations. This is [easily mitigated with query style](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/get-started/azure-integration-metrics/#metrics-with-many-dimension-combinations).
```

Comment:
```suggestion
* A metric name may appear twice when the metric has different dimension combinations. You can prevent duplicated metrics by [creating a query that filters aggregations of your data](/docs/infrastructure/microsoft-azure-integrations/get-started/azure-integration-metrics/#metrics-with-many-dimension-combinations).
```

Reply:
@akristen Love these improvements. Just one small adjustment.

_> You can prevent duplicated metrics_

You can prevent duplicate metric names

Reply:
Great catch! Updated and pushing this up. Is it good to merge today? @bibliobrisa